cubic-server/chat/Message.cpp

```diff
@@ -34,8 +34,7 @@ nlohmann::json chat::Message::toJson() const
 {
     nlohmann::json response;
 
-    if (_message.size() > 0)
-        response["text"] = _message;
+    response["text"] = _message;
```

Comment:
why ?

Reply:
With the if statement, an empty message will generate the following raw json : 'null', istead of the desired '{"text":""}'